### PR TITLE
Release notes for pre-release epic branches.

### PR DESCRIPTION
**Pull request by jMyles**

### CODE REVIEW

**Comment by jMyles** [comment]
Opened by mistake while transiting branches.

**Comment by KPrasch** [comment]
Well - which is it :-) Open or closed?

File: docs/source/release_notes/pre_release_epics.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+Coffee was the first epic which brought a usable Arrangement API and regarded the Arrangement as the fundamental representation of a relationship between Alice, Ursula, and Bob - now a mainstay of the NuCypher network.
+
+It also included the first seeds of many of the metaphors that have become common vernacular on the team: "dividing" a stake, the "learning loop", "NetworkMiddleware", and node verification.
+
+
+## Kokonusswasser
+28 Sep 2018
+https://github.com/nucypher/nucypher/pull/435
+
+*Named because the team was learning German and drinknig coconut water.*
```
Comment (tuxxy) [inline docs/source/release_notes/pre_release_epics.md]:
```suggestion
*Named because the team was learning German and drinking coconut water.*
```

**Comment by KPrasch** [comment]
I think we decided that those stubs do not qualify as "pre-releases"

**Pull request merged**